types/vuelidate/index.d.ts

```diff
@@ -1,4 +1,5 @@
-// Type definitions for vuelidate 0.7
+// Type definitions for vuelidate 0.7.3
```

Comment:
It was conscious decision to go with major releases and not include the hot fix version.

Is there some API referred which has been introduced by 0.7.3?

Reply:
no - I did want to differentiate from the previous version because changing ```$params: { [attr: string]: any }``` to ```$params: { [attr: string]: Params | null }```, it may mean some existing typescript code will have to change to allow for possible null - i.e. ```$this.$v.$params.between!.min``` - my testing would indicate this is however correct - this value will be null if a non-leaf node higher in a nested validation. I am new to submitting type definitions, and while more technically correct, I am not sure how to indicate this is a 'breaking' type definition

Reply:
Indeed there might be a use-case that isn't Params nor null. I had such a typing here and one of the initial reviewers gave reason to go for any. I need some time back at my computer to trace that back.

Types versioning isn't very strong, didn't came across any so called best practice about it.